ShoppingListApp/Models/Item.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  Item.swift
+//  ShoppingListApp
+//
+//  Created by Sofia on 16/02/2019.
+//  Copyright © 2019 Sofia. All rights reserved.
+//
+
+import UIKit
+
+class Item {
+    struct Count {
+        enum Unit: String {
+            case pieces = "pcs."
+            case kg = "kg."
+            case gr = "g."
+            case litr = "l."
+            case ml = "ml."
+            case bottle = "btl."
+            case packs = "packs"
+        }
+        let number: Double
+        let units: Unit
+    }
+    let title: String
+    let count: Count
```

Comment:
Please consider renaming `Count` -> `Amount`
```suggestion
    let amount: Amount
```

Reply:
Done